.claude/skills/translate-batch/SKILL.md

```
@@ -0,0 +1,277 @@
+git add <競合ファイル>
+git commit --no-edit
+git push origin <PR の head branch>
+```
+
+その後 6-1 に戻る。
+
+### 6-4: マージ後の後始末
+
+- マージ成功を確認したら終了。
+- **main 統合 PR (base=main) をマージしたときは Slack 通知を送る** — `mcp__claude_ai_Slack__slack_send_message` で channel `C0B3K9LTYHW` (#gitlab-handbook-ja) に投稿。`<@U0A6FDDNTME>` 宛にメンションし、PR 番号・URL・CR 対応の概要・マージ済みである旨を簡潔に伝える。
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)